src/Command/Diff.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+/*
+ * This file is part of the Manala package.
+ *
+ * (c) Manala <contact@manala.io>
+ *
+ * For the full copyright and license information, please refer to the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Manala\Command;
+
+use Manala\Env\EnvEnum;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Process\Process;
+
+/**
+ * @author Maxime STEINHAUSSER <maxime.steinhausser@gmail.com>
+ */
+class Diff extends Command
+{
+    /** @var Filesystem */
+    private $fs;
+
+    public function __construct()
+    {
+        parent::__construct('diff');
+
+        $this->fs = new Filesystem();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setDescription('Computes the diff between the current project and the Manala templates.')
+            ->addArgument('cwd', InputArgument::OPTIONAL, 'The path of the application', getcwd())
+            ->addOption('env', null, InputOption::VALUE_OPTIONAL, 'One of the supported environment types', 'symfony');
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $cwd = realpath($input->getArgument('cwd'));
+        $envType = EnvEnum::create($input->getOption('env'));
+
+        if (!is_dir($cwd)) {
+            throw new \RuntimeException(sprintf('The working directory "%s" doesn\'t exist.', $cwd));
+        }
+
+        $originalTemplatePath = MANALA_DIR.'/src/Resources/'.$envType;
```

Comment:
> A new EnvResourcesLocator might be useful for this. Or simply a new method on the EnvType.

Indeed 👍  some existing code and #29 will need it as well so let's add it in another PR